recipes/google-groups/webview.js

```diff
@@ -0,0 +1,22 @@
+module.exports = Ferdium => {
+  // if the user is on gmail's landing page, go to the login page.
+
+  const getMessages = () => {
+    let countImportant = 0;
+    console.info(countImportant);
+    const conversationLinks = document.querySelectorAll('div[aria-label="Conversations"]');
```

Comment:
This will not work for non-English locales.

Reply:
@vraravam Made the changes and pushed.